web/admin/src/App.css

```diff
@@ -362,6 +363,16 @@
   border-radius: 50%;
 }
 
+.button3{
```

Comment:
Let's use a more meaningful class name.

Reply:
Agreed as it's is an image within a button